### PR TITLE
WIP: First aproach without Overengineering S390 hide booting

### PR DESCRIPTION
**Pull request by teclator**
I will add two approach,
this one
and other moving stuf to a specific class: https://github.com/yast/yast-security/pull/33

### CODE REVIEW

File: src/include/security/helps.rb [hunk 1]
```diff
@@ -352,9 +331,47 @@ def initialize_security_helps(include_target)
...
+    end
+
+    def boot_dialog_help
+      help = _(
+        "<p><b><big>Boot Security</big></b></p>\n<p>In this dialog, change various boot settings related to security.</p>"
+      )
+
+      # Boot dialog help 2/4, not shown for s390 architecture
+      if Arch.s390
+        help += _(
```
Comment (mchf) [inline src/include/security/helps.rb]: what about ```<<``` instead of ```+=``` ?

Comment (teclator) [inline src/include/security/helps.rb]: I did so in the closed one, but @kobliha wasn't sure about some kind of problems, so i replaced but maybe he mind other thing in the comment and not about the operator.

Comment (mvidner) [inline src/include/security/helps.rb]: @mchf We used `<<` in the past but then discovered that if you run in a non-English locale, gettext will give you immutable strings. :bomb:

Comment (teclator) [inline src/include/security/helps.rb]: @mvidner thnx for the explanation!

**Pull request merged**